recipe/build.sh

```diff
@@ -26,6 +26,7 @@ cp $BUILD_PREFIX/share/libtool/build-aux/config.* ./libcharset/build-aux
             --disable-rpath
 
 if [[ "${target_platform}" == osx-* ]]; then
+    make -f Makefile.devel CC="${CC}" CFLAGS="${CFLAGS}" totally-clean
```

Comment:
Do we need that `totally-clean`?
Is there something in the dist files that clashes with the patch or something?

Reply:
I ran the code locally and it didn't remake some stuff. Happy to try without for now though.